### ISSUE TITLE
Ability to configure sw.js file name in replayBase or similar

### ISSUE THREAD

**Issue by RafaPolit** [webrecorder/replayweb.page#160]
Good morning,

We are trying to implement ReplayWeb on a system that serializes any uploaded file. Therefore, we can upload the `sw.js` file, but it gets renamed to something like `ads8f798798ads98.js`.  It would be ideal if, together with the option to configure the replayBase URL for the file, we could actually tell the service the name of the file.  Something like: the `sw.js` file is actually located at and named `xxx/yyy/ads8f798798ads98.js`

Thanks for any insight if this is feasible.

**Comment by ikreymer** [comment]
Yeah, i think that could be support, perhaps `serviceWorkerURL`, which could be `serviceWorkerURL=xxx/yyy/ads8f798798ads98.js` and then replayBase would be set to `xxx/yyy`.
Are you expecting this URL, and in particular, the path to change a lot? Ideally, it will remain in a fixed location.

Also, I'm curious what the system is that you're integrating replayweb.page with!

**Comment by makew0rld** [comment]
I am also trying to use replaywebpage on the Uwazi platform and would appreciate setting a custom service worker path.
